edx_sga/static/css/edx_sga.css

```diff
@@ -119,7 +121,32 @@
 }
 
 .modal-type-edx_sga ul.editor-modes.action-list.action-modes li a:hover {
-   background: #777;
-   color: #FFF;
+    background: #777;
+    color: #FFF;
+}
+
+.sga-block a.button {
+    background-color: #E5E4E3;
+    border: 1px solid #cec5c5;
+    border-radius: 2px;
+    text-decoration: none;
+    -moz-user-select: none;
+    box-sizing: border-box;
+    cursor: pointer;
+    display: inline-block;
+    line-height: normal;
+    text-align: center;
+    vertical-align: middle;
+    white-space: nowrap;
+    top: 0px;
+    left: 0px;
+    z-index: 2;
+    padding: 2px;
+    width: 200px;
+    color: #333346;
+    font-weight: bold;
+    font-size: 0.8125em;
+    margin-top: 1px;
```

Comment:
This makes the button one pixel lower than the other button.  I think that if you remove this line this is definitely good to go.  Sorry to be so particular:
![image](https://cloud.githubusercontent.com/assets/596029/6835561/0556a862-d312-11e4-8c23-033d4c3dd9c4.png)